src/iguana/algorithms/clas12/ZVertexFilter/Config.yaml

```
@@ -3,12 +3,15 @@ clas12::ZVertexFilter:
 
   # default cuts
   - default:
-    cuts: [ -20.0, 20.0 ]
+    pids: [ 11 ]
+    cuts: [ -20, 20 ]
```

Just to be safe, make sure they're not parsed as integers.
```suggestion
    cuts: [ -20.0, 20.0 ]
```